REFERENCE.md

```diff
@@ -6,7 +6,7 @@
 
 ### Classes
 
-* [`hiera`](#hiera): == Class: hiera  This class handles installing the hiera.yaml for Puppet's use.  === Parameters:    See README.  === Actions:  Installs eithe
+* [`hiera`](#hiera): This class handles installing the hiera.yaml for Puppet's use.
 * [`hiera::deep_merge`](#hiera--deep_merge): == Class: hiera::deep_merge  This class installs and configures deep_merge  === Authors:  Joseph Yaworski <jyaworski@carotid.us>  === Copyrig
 * [`hiera::eyaml`](#hiera--eyaml): == Class: hiera::eyaml  This class installs and configures hiera-eyaml  === Authors:  Terri Haber <terri@puppetlabs.com>  === Copyright:  Cop
 * [`hiera::eyaml_gpg`](#hiera--eyaml_gpg): == Class hiera::eyaml_gpg  This calls install and configures hiera-eyaml-gpg
```

Comment:
I see a lot of `===` nonsense.  Do you plan to fix these in this PR are focus on init.pp for now?

Reply:
I was focusing on the warnings the validate rake task produces. But I can fix the legacy stuff here too.

Reply:
btw: how do we handle the === Copyright sections? write copyright into LICENSE and delete it from the classes? Or put it in the README?

Reply:
Ok, I am gonna call it here for the copyright stuff. I am going to leave that in, we can fix those at a later date I believe.